### PR TITLE
Add submission success page

### PR DESCRIPTION
**Pull request by paulopmt1**
#### Changes proposed in this Pull Request

* Added a new submission success page on `http://calypso.localhost:3000/marketplace/submission-success` route.
* Reused MasterbarStyled from the `http://calypso.localhost:3000/marketplace/thank-you/woocommerce-subscriptions/politicaemvideira.com` screen
#### Testing instructions

* Open the page http://calypso.localhost:3000/marketplace/thank-you/woocommerce-subscriptions/politicaemvideira.com and make sure the back button still works as expected
* Design should match the Figma file: SJ6I0rUUtHNdHMF2KMMJig-fi-311%3A5435
* Open the page http://calypso.localhost:3000/marketplace/submission-success and you should see this screen:

image

* It should work in all supported browsers
* It should work on mobile

Related to https://github.com/Automattic/wp-calypso/issues/63554

### CODE REVIEW

File: client/my-sites/marketplace/pages/submission-success/signup-success.tsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+					<p>
+						{ translate(
+							'Thank you for apply to join the WordPress.com marketplace. We’ll be in touch with you very soon to discuss next steps.'
+						) }
+					</p>
+				</div>
+				<div className="signup-success__body">
+					<div className="signup-success__row">
+						<div className="signup-success__row-icon">
+							<WelcomeCartIcon />
```
Comment (lsl) [inline client/my-sites/marketplace/pages/submission-success/signup-success.tsx]: Should use gridicons for these http://automattic.github.io/gridicons/

```suggestion
							<Gridicon icon="cart" />
```
cc @SaxonF

File: client/my-sites/marketplace/pages/submission-success/signup-success.tsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+							<p>{ translate( 'Read more about selling on the WordPress.com marketplace.' ) }</p>
+						</div>
+						<Button isSecondary href="https://developer.wordpress.com/wordpress-com-marketplace/">
+							{ translate( 'Continue' ) }
+						</Button>
+					</div>
+					<hr />
+					<div className="signup-success__row">
+						<div className="signup-success__row-icon">
+							<WelcomeConfigIcon />
```
Comment (lsl) [inline client/my-sites/marketplace/pages/submission-success/signup-success.tsx]:
```suggestion
							<Gridicon icon="cog" />
```

**Review by lsl — APPROVED** [review]
LGTM, but please replace SVG's with gridicons before merging.

**Comment by lsl** [comment]
Could probably ship this as is though there are some console warnings, looks like something to do with the SVG styling.

**Comment by paulopmt1** [comment]
Thanks for reviewing @lsl and replacing the svg's with gridicons. I'll replace the SVG for a correspondent PNG image, so we don't need to style it (changing dash-case to camelCase attributes) and as we can see, the quality becomes better by just copying and pasting the image:

image

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7314271

Thank you @paulopmt1 for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7314271).

**Pull request merged**